File: Code_samples/C#/Fhi.OpenApiClient/Models/DimensionsOpenResponse.cs
namespace Fhi.OpenApiClient.Models
{
    internal class DimensionsOpenResponse
    {
        public Dimension[]? Dimensions { get; set; }
    }

    internal class Dimension
    {
        public string? Code { get; set; }
        public string? Label { get; set; }
        public IEnumerable<Category>? Categories { get; set; }
    }

    public class Category
    {
        public string? Label { get; set; }
        public string? Value { get; set; }
        public IEnumerable<Category>? Children { get; set; }
    }
}

File: Code_samples/Kotlin/FhiOpenApiClient/src/main/kotlin/models/DimensionsOpenResponse.kt
package models

import kotlinx.serialization.Serializable

@Serializable
internal class DimensionsOpenResponse {
    var dimensions: Array<Dimension>? = null;

    @Serializable
    internal class Dimension
    {
        var code: String? = null;
        var label: String? = null;
        var categories: Array<Category>? = null;
    }

    @Serializable
    internal class Category
    {
        var label: String? = null;
        var value: String? = null;
        var children: Array<Category>? = null;
    }
}